vscode-extension/package.json

```diff
@@ -246,6 +256,12 @@
         },
         {
           "command": "vscode-aiconfig.createExampleAIConfigYAML"
+        },
+        {
+          "command": "vscode-aiconfig.createEmptyAIConfigYAML"
```

Comment:
One of these should be JSON.

Also, if order of these matters we should do Empty before Example and YAML before JSON

Reply:
ordering matters above in the section of
```
"contributes": {
    "commands": [
```

With that in mind I don't know what this section does. In the testplan of this diff the 4 create commands work as intended.

Reply:
but I'l add it in

Reply:
synced offline with @rossdanlm , this is for the new file flow within vscode